website/docs/server.md

```diff
@@ -123,6 +144,31 @@ Logout a user and invalidate their session with the session access token.
 await accountsServer.logout(accessToken);
 ```
 
+### Check if session is impersonated
```

Comment:
What do you think about moving the "Authorize to impersonate" paragraph at the same level as here?

Reply:
Moved it.